pkg/collector/py/loader.go

```diff
@@ -52,19 +52,37 @@ func NewPythonCheckLoader() (*PythonCheckLoader, error) {
 func (cl *PythonCheckLoader) Load(config check.Config) ([]check.Check, error) {
 	checks := []check.Check{}
 	moduleName := config.Name
+	whlModuleName := fmt.Sprintf("datadog.%s.%s", config.Name, config.Name)
```

Comment:
Shouldn't (for example) `datadog.ntp` be enough? Why do we try to import `datadog.ntp.ntp`?

Reply:
If I only import `datadog.{integratation}` then this line:
```
checkClass, err := findSubclassOf(cl.agentCheckClass, checkModule, glock)
```
fails to find the subclass. I can try to dig deeper, but this just worked, so I kept it.

Reply:
I believe that's because the `__init__.py` in the check package should contain `from .whatever import WhateverCheck`. I'd rather document this as a requirement than force packages to have one python module named after the check (like `datadog/ntp/ntp.py`).